index.js

```diff
@@ -10,4 +10,104 @@ const menuBackground = document.getElementById('menuBackground');
+  clonedHeader2.classList.add('cloned-header');
+  const Bclose = document.createElement('img');
+  Bclose.src = 'images/bclose.png';
+  Bclose.classList.add('b-close');
+  const clonedSubheader = popUp.subHeader[c].cloneNode(true);
+  clonedSubheader.classList.add('cloned-subheader');
+  const clonedImage = popUp.image[d].cloneNode(true);
+  clonedImage.classList.add('cloned-img');
+  const clonedDesktopImage = popUp.image[e].cloneNode(true);
+  clonedDesktopImage.classList.add('cloned-desk-img');
+  const popInfoContainer = document.createElement('div');
+  const popBodyContainer = document.createElement('div');
+  popBodyContainer.classList.add('pop-body-container');
+  popInfoContainer.classList.add('pop-info-container');
+  const description = document.createElement('p');
+  description.innerHTML = popUp.description;
+  description.classList.add('pop-description');
+  const clonedTech = popUp.technologies[f].cloneNode(true);
+  clonedTech.setAttribute('id', 'cloned-tech');
+  const githubTech = document.createElement('li');
+  githubTech.innerHTML = 'Github';
+  githubTech.classList.add('desktop');
+  const rubyTech = document.createElement('li');
+  rubyTech.innerHTML = 'Ruby';
+  rubyTech.classList.add('desktop');
+  const bootTech = document.createElement('li');
+  bootTech.innerHTML = 'Bootstrap';
+  bootTech.classList.add('desktop');
+  const buttonBox = document.createElement('div');
+  buttonBox.classList.add('pop-button-box');
+  const liveButton = document.createElement('button');
+  const sourceButton = document.createElement('button');
+  liveButton.innerHTML = popUp.linkLive;
+  sourceButton.innerHTML = popUp.linkSource;
+  liveButton.classList.add('pop-button');
+  sourceButton.classList.add('pop-button');
+  const liveIcon = document.createElement('img');
+  liveIcon.src = 'images/live.svg';
+  liveIcon.classList.add('pop-button-icon');
+  const gitIcon = document.createElement('img');
+  gitIcon.src = 'images/github-blue.svg';
+  gitIcon.classList.add('pop-button-icon');
+  document.body.appendChild(backPop);
+  backPop.appendChild(popCard);
+  popCard.appendChild(headerBox);
+  headerBox.appendChild(clonedHeader);
+  headerBox.appendChild(clonedHeader2);
+  headerBox.appendChild(Bclose);
+  popCard.appendChild(clonedSubheader);
+  popCard.appendChild(clonedImage);
+  popCard.appendChild(clonedDesktopImage);
+  popCard.appendChild(popBodyContainer);
+  popBodyContainer.appendChild(description);
+  popBodyContainer.appendChild(popInfoContainer);
+  popInfoContainer.appendChild(clonedTech);
+  popInfoContainer.appendChild(buttonBox);
+  clonedTech.appendChild(githubTech);
+  clonedTech.appendChild(rubyTech);
+  clonedTech.appendChild(bootTech);
+  buttonBox.appendChild(liveButton);
+  buttonBox.appendChild(sourceButton);
+  liveButton.appendChild(liveIcon);
+  sourceButton.appendChild(gitIcon);
+
+  function closePopUp() {
+    backPop.remove();
+  }
+  Bclose.addEventListener('click', closePopUp);
```

Comment:
- [ ] As I can see here you used this way to add the html pop up but I think it would be better if you use a loop then you can make inner html inside you javscript to make sure you follow best javascript practice and to make your code dry and clean so kindly use the innerhtml way it would be more organized 💯💯🙏🏻🙏🏻